domain-server/resources/web/js/shared.js

```diff
@@ -230,213 +230,212 @@ function getDomainFromAPI(callback) {
 
 function chooseFromHighFidelityPlaces(accessToken, forcePathTo, onSuccessfullyAdded) {
   if (accessToken) {
+      getMetaverseUrl(function(metaverse_url) {
+
+        var loadingDialog = showLoadingDialog(Strings.ADD_PLACE_LOADING_DIALOG);
+
+        function loadPlaces() {
+          $.ajax("/api/places", {
+            dataType: 'json',
```

Comment:
There are only two lines of code actually changed in here and below, but the indentation has been partly changed from the file's default of 2 characters which has made indentation inconsistent and thrown off the diff. I suggest fixing up indent to be 2 characters to make the file consistent and make the changes able to be seen.

Reply:
So I fixed the indentation but it will still look weird as I indented it all anyway, it's just consistent now.

And yes, it is a very minimal amount of lines changed. If any. The code in there is all quite messy.